tools/eslint-rules/eslint-check.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @fileoverview Check that common.skipIfEslintMissing is used if
+ * @file Check that common.skipIfEslintMissing is used if
  *               the eslint module is required.
```

Comment:
```suggestion
 *       the eslint module is required.
```

Reply:
I 100% agree but it's against our CONTRIBUTING.md, since it's allowed in our formatter and linter right now. This should be handled through a formatter. There is a particular sentence in the document about: "Anything that can be automated should be automated"

Reply:
Wouldn't `jsdoc/check-line-alignment` enforce that?

Reply:
I enabled it but it didn't fix this.